src/promptflow/promptflow/contracts/tool.py

```diff
@@ -252,6 +252,10 @@ class InputDefinition:
     # For a custom strong type connection input, the type should be 'CustomConnection',
     # while the custom_type should be the custom strong type connection class name.
     custom_type: List[str] = None
+    # Param 'index' is used for UI to render tool inputs. Currently, it contains two properties:
```

Comment:
please make the comment more clear by explaining what is ui_hints, what ui_hints include, and why need to add this property.

Reply:
No need to change tool contract for now, reverted these changes, thanks.